build/templates/functions.rst/lock_session_wrapper.rst.mako

```diff
@@ -0,0 +1,26 @@
+<%page args="function, config, method_template, indent"/>\
+<%
+    import build.helper as helper
+%>\
+.. py:method:: ${function['python_name']}()
+
+    | Obtains a multithread lock on the device session. Before doing so, the
+      software waits until all other execution threads release their locks
+      on the device session.
+    | Other threads may have obtained a lock on this session for the
+      following reasons:
+
+    -  The application called the :py:meth:`nidcpower.Session.lock_session` method.
+    -  A call to NI-DCPower locked the session.
```

Comment:
Why is NI-DCPower mentioned here? Seems like you should use the mako variables.

There's a few more places with same issue.